features/shared/environment.steps.ts

```diff
@@ -1,11 +1,27 @@
-import {Given, After} from '@cucumber/cucumber';
+import {execSync} from 'child_process';
+
+import {Given, After, Then} from 'cucumber';
 import tmp from 'tmp';
 import rimraf from 'rimraf';
+import path from 'pathe';
+import fs from 'fs-extra';
 
 Given('I have a working directory', function () {
   this.temporaryDirectory = tmp.dirSync().name;
 });
 
+Given(/I have the fixture (.+) in the working directory/, function (fixture) {
+  const from = path.join(__dirname, '../../fixtures', fixture);
+  const to = this.temporaryDirectory;
+  fs.copySync(from, to);
+});
+
+Then('I install the fixture dependencies', function () {
+  execSync('pnpm install', {stdio: 'inherit', cwd: this.temporaryDirectory});
+});
+
 After(async function () {
-  rimraf.sync(this.temporaryDirectory);
```

Comment:
Today I learnt rimraf is The UNIX command rm -rf for node.
[rimraf](https://www.npmjs.com/package/rimraf)

Reply:
Yes! 😁  

Indeed, it is a package that tries to make the `rm -rf` behavior consistent between different operating systems (e.g. in Window OS the command to remove recursively is different. IIRC, it is `del <something>`). As Node aims to have multi-platform support, there was a need to use a command that is cross-platform to clean up the build artifacts (e.g. transpiler/bundler output) so they came up with this.

Reply:
[pathe](https://www.npmjs.com/package/pathe) and [shelljs](https://www.npmjs.com/package/shelljs) aim to do the same for path manipulation and process execution